sylvia-derive/src/message.rs

```diff
@@ -728,7 +730,7 @@ impl<'a> GlueMessage<'a> {
             quote! { #variant(#module :: #name<#(#generics,)*>) }
         });
 
-        let impl_msg_name = match msg_ty {
+        let impl_msg = match msg_ty {
             MsgType::Exec => Ident::new("ImplExecMsg", Span::call_site()),
             MsgType::Query => Ident::new("ImplQueryMsg", Span::call_site()),
```

Comment:
I will do it under another review as it would require renaming all of the variables like impl_msg_name and so on and I don't really want to get into possible name collissions here.